docs/versioned/eventing/concepts/event-mesh.md

```diff
@@ -13,13 +13,13 @@ In an Event Mesh, both producing and consuming applications do not need to imple
 
 ## Knative Event Mesh
 
-The above mentioned _event brokers_ map directly to a core API in Knative Eventing: the [`Broker` API](../brokers) offers a discoverable endpoint for event ingress and the [`Trigger` API](../triggers) completes the offering with its event filtering and delivery capabilities.  With these APIs Knative Eventing offers an Event Mesh as defined above:
+The above mentioned _event brokers_ map directly to a core API in Knative Eventing: the [`Broker` API](../brokers) offers a discoverable endpoint for event ingress and the [`Trigger` API](../triggers/) completes the offering with its event filtering and delivery capabilities.  With these APIs Knative Eventing offers an Event Mesh as defined above:
```

Comment:
These links aren't working properly -- I suspect you need:


```suggestion
The above mentioned _event brokers_ map directly to a core API in Knative Eventing: the [`Broker` API](../brokers/README.md) offers a discoverable endpoint for event ingress and the [`Trigger` API](../triggers/README.md) completes the offering with its event filtering and delivery capabilities.  With these APIs Knative Eventing offers an Event Mesh as defined above:
```

Reply:
(This is where actually linking to the `.md` files rather than to the directory helps both mkdocs and tools like VSCode)